1-trackers/README.md

```diff
@@ -4,59 +4,29 @@
-| [arduino-tracker][arduino]       | An event tracker for IP-connected Arduino boards               | Production-ready |
-| [dotnet-tracker][dotnet]         | An event tracker for the .NET platform                         | Production-ready |
-| [ios-tracker][ios]               | An event tracker for iOS apps and games                        | Production-ready |
-| [java-tracker][java]             | An event tracker for Java-based desktop and server apps, servlets and games | Production-ready |
-| [javascript-tracker][javascript] | A client-side JavaScript tracker for web browser use           | Production-ready |
-| [lua-tracker][lua]               | An event tracker for Lua apps, games and plugins               | Production-ready |
-| [no-js-tracker][no-js]           | A pixel-based tracker for no-JavaScript web environments       | Production-ready |
-| [nodejs-tracker][nodejs]         | An event tracker for node.js apps and servers                  | Production-ready |
-| [php-tracker][php]               | An event tracker for PHP apps and scripts                      | Production-ready |
-| [python-tracker][python]         | An event tracker for Python and Django webapps, apps and games | Production-ready |
-| [ruby-tracker][ruby]             | An event tracker for Ruby and Rails apps and gems              | Production-ready |
-| [scala-tracker][scala]           | An event tracker for Scala, Akka and Play apps and servers     | Production-ready |
-| [unity-tracker][unity]           | An event tracker for Unity games and apps                      | Beta             |
-
-For other trackers (e.g. PHP, Go) and their approximate timelines, please see the [Product Roadmap][roadmap].
-
-## Find out more
-
-| Technical Docs               | Setup Guide           | Roadmap & Contributing               |         
-|------------------------------|-----------------------|--------------------------------------|
-| ![i1][techdocs-image]       | ![i2][setup-image]   | ![i3][roadmap-image]                |
-| [Technical Docs][tech-docs] | [Setup Guide][setup] | _coming soon_                        |
+The up-to-date list of trackers and information about their setup can be found on our [docs website][docs-website].
+
+## Github repos
+
+- [Android](https://github.com/snowplow/snowplow-android-tracker)
+- [.NET](https://github.com/snowplow/snowplow-dotnet-tracker)
+- [Go](https://github.com/snowplow/snowplow-golang-tracker)
+- [JAVA](https://github.com/snowplow/snowplow-java-tracker)
+- [Javascript](https://github.com/snowplow/snowplow-javascript-tracker)
+- [Node.js](https://github.com/snowplow/snowplow-nodejs-tracker)
+- [Objective-C](https://github.com/snowplow/snowplow-objc-tracker)
+- [PHP](https://github.com/snowplow/snowplow-php-tracker)
+- [Python](https://github.com/snowplow/snowplow-python-tracker)
+- [Ruby](https://github.com/snowplow/snowplow-ruby-tracker)
```

Comment:
I probably could've just done this once, but I've started so...
Missing React Native Tracker: https://github.com/snowplow-incubator/snowplow-react-native-tracker